src/hotspot/cpu/aarch64/globals_aarch64.hpp

```diff
@@ -110,7 +110,10 @@ define_pd_global(intx, InlineSmallCode,          1000);
   product(int, SoftwarePrefetchHintDistance, -1,                        \
           "Use prfm hint with specified distance in compiled code."     \
           "Value -1 means off.")                                        \
-          range(-1, 4096)
+          range(-1, 4096)                                               \
+  product(ccstr, UsePauseImpl, "none",                                  \
```

Comment:
The name "UsePauseImpl" fails to make the connection with `onSpinWait`. If you called it something like `OnSpinWaitImpl` that would make the connection.

Reply:
Done